services/sessionservice/session.go

```diff
@@ -245,8 +244,8 @@ func CreateAndSetAccessTokenCookie(w http.ResponseWriter, sessionUUID string) {
 		Path:     "/",
 		Value:    sessionUUID,
 		HttpOnly: true,
-		SameSite: http.SameSiteNoneMode, // change to http.SameSiteStrictMode in prod
-		Secure:   false,                 // change to true in prod
+		SameSite: http.SameSiteNoneMode, // TODO change to http.SameSiteStrictMode in prod
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.